src/shared/components/Typography/Typography.tsx

```diff
@@ -0,0 +1,26 @@
+import { theme } from '../../theme';
+import { Body1, Body2, H1, H2, H3 } from './styles';
+import { TypographyProps } from './types';
+
+export const Typography = ({
```

Comment:
**Question:** I've noticed that for Button, Input, and Card you used named function and here is a function expression. What is your preferred pattern for this case?